src/elf-council-tokenlist/testnet.tokenlist.json

```diff
@@ -73,7 +73,7 @@
       "symbol": "",
       "extensions": {
         "coreVoting": "0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9",
-        "votingPowerBound": "100",
+        "votingPowerBound": "100000000000000000000",
```

Comment:
seems suspect? 😄

Reply:
no its right, should be 100 eth value, not 100 wei value.